discovery/gossiper_test.go

```diff
@@ -63,6 +63,8 @@ var (
 	retransmitDelay     = time.Hour * 1
 	rebroadcastInterval = time.Second * 100
 	proofMatureDelta    uint32
+
+	testTimestamp = uint32(time.Now().Unix())
```

Comment:
Any reason this always needs to be the current timestamp? IIRC, @joostjager has had some remarks on using dynamic values within tests.

Reply:
I indeed use static time values in unit tests to keep them deterministic.

Reply:
Nice, will see if I can do the same! 😀

Reply:
Done!